sycl/include/sycl/ext/oneapi/experimental/annotated_ptr/annotated_ptr.hpp

```diff
@@ -197,10 +197,6 @@ __SYCL_TYPE(annotated_ptr) annotated_ptr<T, detail::properties_t<Props...>> {
   static constexpr bool is_valid_property_list =
       is_property_list<property_list_t>::value;
   static_assert(is_valid_property_list, "Property list is invalid.");
-  static constexpr bool contains_valid_properties =
```

Comment:
I think we need to fix this function and improve the error message instead of removing it here